pkg/services/apiserver/aggregator/aggregator.go

```diff
@@ -127,8 +127,9 @@ func CreateAggregatorConfig(commandOptions *options.Options, sharedConfig generi
 			ClientConfig:          sharedConfig.LoopbackClientConfig,
 		},
 		ExtraConfig: aggregatorapiserver.ExtraConfig{
-			ProxyClientCertFile: commandOptions.KubeAggregatorOptions.ProxyClientCertFile,
-			ProxyClientKeyFile:  commandOptions.KubeAggregatorOptions.ProxyClientKeyFile,
+			DisableRemoteAvailableConditionController: true,
```

Comment:
Just checking, isn't this going to break cloud APIs aggregation?

Reply:
Actually, not. We anyway use our fork of RemoteAvailableCondition controller, the new`k8s.io/aggregator` comes with this new bool `DisableRemoteAvailableConditionController` which allows us to select ours and not theirs more straightforwardly. We haven't used theirs anyway as it requires corev1 Service while ours uses service.grafana.app/ExternalName.

Reply:
Since theirs didn't have a way of disabling until very recently, providing them a fake client set has only been the way of making in ineffectual. This boolean is a more succinct way of opting out.

Reply:
Ephemeral instances are working fine:

```
kubectl get apiservices
v1alpha1.gamma.ext.grafana.com                stacks-5865/v1alpha1.gamma.ext.grafana.com          True        3m22s
v1alpha1.servicemodel.ext.grafana.com         stacks-5865/v1alpha1.servicemodel.ext.grafana.com   True        3m22s
```